README.md

```diff
@@ -33,17 +33,37 @@ and have a [Code of
 Conduct](http://plasmapy.readthedocs.io/en/master/CODE_OF_CONDUCT.html).
 New contributors are very welcome!
 
+# Feedback and communication
+
+## [Matrix chat](https://riot.im/app/#/room/#plasmapy:matrix.org)
+
 If you have any questions, the quickest way to get a response is to ask
 on our
 [Matrix](https://riot.im/app/#/room/#plasmapy:matrix.org)/[Gitter](https://gitter.im/PlasmaPy/Lobby)
-channel.  For a slower response, you may also contact Nick Murphy at
-<namurphy@cfa.harvard.edu> and Drew Leonard at
```

Comment:
That reminds me that we need to set up a main email address for PlasmaPy, though no need to worry about that right now.

Reply:
I just noticed your comment above.  Hm...we do have a plasmapy-dev list that we rarely use.  Maybe it could go there.  Alternatively, we could create an email list that is primarily for the Coordinating Committee.

Reply:
Is plasmapy-dev the same thing as the google groups thing at https://groups.google.com/forum/#!forum/plasmapy ? Either way, that probably wouldn't work unless it has an associated email address (which I haven't been able to find if it exists). The way this works is that I can add people (via email addresses) to collaborators and those people can then see the responses.

Ideally, we'd eventually automate crossposting those to GitHub or the mailing list.